src/database.ts

```diff
@@ -0,0 +1,10 @@
+export const books = [
```

Comment:
```suggestion
export const mockedBooks = [
```
ou se já tiver uma função como `getBooks`, ai seria
```ts
getBooks(): Promise<Book[]> {
  return Promise.resolve([
    { id: "1", name: "book1" },
    { id: "2", name: "book2" },  
    // ...
  ]);
}